runtime/src/main/java/org/corfudb/runtime/view/ObjectBuilder.java

```diff
@@ -46,6 +46,9 @@
     @Setter
     ISerializer serializer = Serializers.JSON;
 
+    @Setter
+    boolean needOrderedUUIDs;
```

Comment:
This seems to be specific to the object type itself. Needs to be pushed to the constructor.

Reply:
Done. I will push this out to the constructor as you suggest.